packages/openshift/result.py

```diff
@@ -70,6 +75,14 @@ def as_json(self, indent=4, truncate_stdout=-1, redact_tokens=True, redact_strea
 
     def add_action(self, action):
         self.__actions.append(action)
+        if self.limit_tracking_actions is not None and self.actions_count() > self.limit_tracking_actions:
+            self.pop_first_action()
+
+    def actions_count(self):
+        return len(self.__actions)
+
+    def pop_first_action(self):
```

Comment:
No need for these two methods (`actions_count` and `pop_first_action`) any more. It doesn't save code and doesn't expose anything we actually want the user of the object actually doing.